README.md

```diff
@@ -13,37 +13,6 @@ The JS SDK is used as a way to share common code for platform specific SDK shims
 
 Refer to the Kinvey [DevCenter](http://devcenter.kinvey.com/) for documentation on using Kinvey.
 
-## How to use
```

Comment:
@thomasconner I understand why this was removed, but how about leaving a couple of notes about heading to the correct SDK and following the instructions on the README there?